site-service/src/main/java/uk/ac/ox/ndph/mts/site_service/repository/FhirRepository.java

```diff
@@ -42,18 +42,16 @@
      * @return all organization instances in the store, might be empty, not null
      */
     List<Organization> findOrganizations();
-
+    
```

Comment:
This just seems like a spurious commit, no substantive changes - suggest reverting it.